app.js

```diff
@@ -0,0 +1,321 @@
+            animationEnabled: true,
+            title: { text: "ערך מטבעות נבחרים ב-USD" },
+            axisX: { valueFormatString: "HH:mm:ss" },
+            axisY: { title: "Coin Value (USD)", prefix: "$" },
+            legend: { cursor: "pointer", verticalAlign: "top" },
+            data: state.selectedForReport.map(symbol => ({
+                type: "line",
+                name: symbol,
+                showInLegend: true,
+                xValueType: "dateTime",
+                dataPoints: [] 
+            }))
+        };
+        const chart = new CanvasJS.Chart("chartContainer", chartOptions);
+        chart.render();
+
+        
+        async function updateChart() {
+            try {
+                const symbols = state.selectedForReport.join(',');
+                console.log("Symbols to fetch:", symbols);
+                const response = await fetch(`${API_URLS.multiPrice}?fsyms=${symbols}&tsyms=USD`);
+                const prices = await response.json();
+
+               
+                if (prices.Response === "Error" || Object.keys(prices).length === 0) {
+                    console.error("API response error or empty:", prices.Message);
+                    return;
+                }
+
+                const time = new Date().getTime();
+
+               
+                chart.options.data.forEach(series => {
+                    const symbol = series.name;
+                    if (prices[symbol] && prices[symbol].USD) {
+                        series.dataPoints.push({ x: time, y: prices[symbol].USD });
+                    }
+                });
+                chart.render();
+            } catch (error) {
+                console.error("Chart update failed:", error);
+                stopLiveReports();
+            }
+        }
+
+        updateChart();
+        state.liveReportInterval = setInterval(updateChart, 2000); 
+    }
+
+    function stopLiveReports() {
+        if (state.liveReportInterval) {
+            clearInterval(state.liveReportInterval);
+            state.liveReportInterval = null;
+        }
+    }
+
+
+    
+
+    function showLoader(show) {
+        elements.loader.style.display = show ? "block" : "none";
+        elements.coinsContainer.style.display = show ? "none" : "flex";
+    }
+
+    function setupEventListeners() {
+        elements.coinsContainer.addEventListener("click", (event) => {
+            const target = event.target;
+            if (target.classList.contains("more-info-btn")) {
+                handleMoreInfoClick(target.dataset.coinId);
+            }
+        });
+
+        elements.coinsContainer.addEventListener("change", (event) => {
+            const target = event.target;
+            if (target.classList.contains("report-toggle")) {
+                handleToggleChange(target.dataset.coinSymbol, target.checked, target);
+            }
+        });
+
+        elements.searchForm.addEventListener("submit", (event) => {
+            event.preventDefault();
```

Comment:
and what if i search when in `about`?
<img width="1776" height="463" alt="Screenshot 2025-09-10 at 21 00 18" src="https://github.com/user-attachments/assets/37bc77ec-0875-426b-b9f7-9fb5b66eb82a" />